docs/components/App.js

```diff
@@ -12,6 +12,7 @@ type Props = {|
 
 export default function App({ children }: Props): Node {
   const router = useRouter();
+  const [isHomePage, setIsHomePage] = useState(false);
```

Comment:
since `useRouter` is a hook, and it might cause a re-render on change,  maybe you could just directly use it as a direct variable, instead of setting `isHomePage` as state variable

```jsx
 const router = useRouter();
 const isHomePage = router.pathName === 'home';
```